hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -834,18 +850,22 @@ genCloseTx numParties = do
   ctx <- genHydraContextFor numParties
   (u0, stOpen) <- genStOpen ctx
   snapshot <- genConfirmedSnapshot 0 u0 (ctxHydraSigningKeys ctx)
-  pointInTime <- genPointInTime
+  let (UnsafeContestationPeriod cp) = ctxContestationPeriod ctx
+      contestationPeriodSlotNo = fromIntegral cp
```

Comment:
This is an *unsafe* conversion of seconds to slots!! It's even wrong for our devnet configuration!